src/main/java/mekanism/common/block/BlockBasic.java

```diff
@@ -449,7 +447,7 @@ else if(tile instanceof TileEntityBin)
 
 						if(entityplayer.isSneaking())
 						{
-							dismantleBlock(state, world, pos, false);
+							dismantleBlock(state, world, pos);
```

Comment:
why is there a new `dismantleBlock` method?

Reply:
We don't need neither the ```returnBlock``` boolean anymore nor reinventing the wheel while dropping blocks with the other changes.